certified-connectors/RecordedFutureNew/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1575 @@
+                                        "intelCard":{
+                                            "type":"string",
+                                            "description":"Recorded Future Intelligence Card Link",
+                                            "title":"intelCard",
+                                            "x-ms-visibility":"important"
+                                        },
+                                        "risk":{
+                                            "type":"object",
+                                            "properties":{
+                                                "criticalityLabel":{
+                                                    "type":"string",
+                                                    "description":"Recorded Future Indicator Criticality Level",
+                                                    "title":"criticalityLabel",
+                                                    "x-ms-visibility":"important"
+                                                },
+                                                "score":{
+                                                    "type":"integer",
+                                                    "format":"int32",
+                                                    "description":"Recorded Future Indicator Risk Score",
+                                                    "title":"score",
+                                                    "x-ms-visibility":"important"
+                                                },
+                                                "evidenceDetails":{
+                                                    "type":"array",
+                                                    "items":{
+                                                        "type":"object",
+                                                        "properties":{
+                                                            "mitigationString":{
+                                                                "type":"string",
+                                                                "description":"mitigationString",
+                                                                "x-ms-visibility":"internal"
+                                                            },
+                                                            "timestamp":{
+                                                                "type":"string",
+                                                                "description":"timestamp",
+                                                                "x-ms-visibility":"internal"
+                                                            },
+                                                            "criticalityLabel":{
+                                                                "type":"string",
+                                                                "description":"criticalityLabel",
```

Comment:
@johannesh-recordedfuture All the description values should be free from grammatical and spell errors. Please provide all the valid descriptions.